shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared;
+
+import com.google.common.base.Preconditions;
+
+public class MetricsTags {
+
+    // Metric Tag Names
+    public static final String TAG_CONTAINER = "container";
+    public static final String TAG_HOST = "host";
+    public static final String TAG_SCOPE = "scope";
+    public static final String TAG_STREAM = "stream";
+    public static final String TAG_SEGMENT = "segment";
+    public static final String TAG_TRANSACTION = "transaction";
+    public static final String TAG_EPOCH = "epoch";
+
+    private static final String TRANSACTION_DELIMITER = "#transaction.";
+    private static final String EPOCH_DELIMITER = ".#epoch.";
+    /**
+     * The Transaction unique identifier is made of two parts, each having a length of 16 bytes (64 bits in Hex).
```

Comment:
A Transaction id is a `UUID` that `represents a 128-bit value`. Then, you can get the most/least significant bytes of the `UUID`, which are 8 bytes (64 bits) each, right?

Reply:
This could be a post-migration enhancement item.